actionview/lib/action_view/railtie.rb

```diff
@@ -37,7 +37,7 @@ class Railtie < Rails::Railtie # :nodoc:
       end
     end
 
-    initializer "action_view.collection_caching" do |app|
+    initializer "action_view.collection_caching", after: "action_controller.set_configs" do |app|
```

Comment:
Theoretically doesn't this introduce coupling between Action View and Action Controller? 😬